tests/helpers.py

```diff
@@ -0,0 +1,241 @@
+    """Instantiate and return a KMD client object."""
+    return kmd.KMDClient(kmd_token, kmd_address)
+
+
+# Env helpers
+
+
+class Account:
+    def __init__(
+        self,
+        address: str,
+        private_key: Optional[str],
+        lsig: Optional[transaction.LogicSig] = None,
+    ):
+        self.address = address
+        self.private_key = private_key
+        self.lsig = lsig
+
+        assert self.private_key or self.lsig
+
+    def mnemonic(self) -> str:
+        return mnemonic.from_private_key(self.private_key)
+
+    def is_lsig(self) -> bool:
+        return bool(not self.private_key and self.lsig)
+
+    @classmethod
+    def create(cls) -> "Account":
+        private_key, address = account.generate_account()
+        return cls(private_key=private_key, address=str(address))
+
+    @property
+    def decoded_address(self):
+        return encoding.decode_address(self.address)
+
+
+def get_kmd_accounts(
+    kmd_wallet_name="unencrypted-default-wallet", kmd_wallet_password=""
+):
+    kmd_client = _kmd_client()
+    wallets = kmd_client.list_wallets()
+
+    walletID = None
+    for wallet in wallets:
+        if wallet["name"] == kmd_wallet_name:
+            walletID = wallet["id"]
+            break
+
+    if walletID is None:
+        raise Exception("Wallet not found: {}".format(kmd_wallet_name))
+
+    walletHandle = kmd_client.init_wallet_handle(walletID, kmd_wallet_password)
+
+    try:
+        addresses = kmd_client.list_keys(walletHandle)
+
+        privateKeys = [
+            kmd_client.export_key(walletHandle, kmd_wallet_password, addr)
+            for addr in addresses
+        ]
+
+        kmdAccounts = [
+            Account(address=addresses[i], private_key=privateKeys[i])
+            for i in range(len(privateKeys))
+        ]
+    finally:
+        kmd_client.release_wallet_handle(walletHandle)
+
+    return kmdAccounts
+
+
+## Teal Helpers
+
+
+def logged_bytes(b: str):
+    return bytes(b, "ascii").hex()
+
+
+def logged_int(i: int):
+    return i.to_bytes(8, "big").hex()
+
+
+def assert_stateful_output(expr: Expr, output: List[str]):
+    assert expr is not None
+
+    src = compile_stateful_app(expr)
+    assert len(src) > 0
+
+    compiled = fully_compile(src)
+    assert len(compiled["hash"]) == 58
+
+    app_id = create_app(
+        compiled["result"],
+        transaction.StateSchema(0, 16),
+        transaction.StateSchema(0, 64),
+    )
+
+    result = call_app(app_id)
+
+    assert result == output
+
+
+def assert_output(expr: Expr, output: List[str], **kwargs):
+    assert expr is not None
+
+    src = compile_app(expr)
+    assert len(src) > 0
+
+    compiled = fully_compile(src)
+    assert len(compiled["hash"]) == 58
+
+    result = execute_app(compiled["result"], **kwargs)
+    assert result == output
+
+
+def compile_app(method: Expr, version: int = 5):
+    return compileTeal(Seq(method, Int(1)), mode=Mode.Application, version=version)
```

Comment:
Shouldn't we simply compile the PyTEAL App program instead of `Seq(method, Int(1))`?

Suggesting new names to def and params to make clear that the result of this compiling is a TEAL source code.
```python
def compile_pyteal_app(program: Expr, version: int = TEAL_VERSION) -> str:
```

Reply:
I'm open to this but since Apps need to return 1 for success,  I wanted to omit as much boiler plate as I could. In this case it just means putting Int(1) on the stack after the method expression

Reply:
So, if I get it correctly, this helper should be renamed into `compile_pyteal_method` right?

Reply:
I'm not sure, I wanted to convey that this creates an application with the method and in the other case that it creates an application that handles create/call/delete separately

Reply:
Just my point of view: I think that compiler helpers should avoid introducing any other PyTEAL logic that wasn't explicitly written in the PyTEAL program source code. I think we should keep compiler helpers domain and PyTEAL domain orthogonal. 

Although I see the practicality of having `Seq(method, Int(1))` I'd rather prefer not to introduce any PyTEAL expression outside what has been defined directly in PyTEAL's domain, to avoid unexpected behaviours (despite the very simple `Expr` addition).

Reply:
Since these are testing helpers I think it makes sense to offer some wrapping of the base functions either directly in this function or a util function that is exported and can provide the wrapping to make it a valid application.